htdocs/nws/clitable.php

```
@@ -97,19 +110,40 @@ function departcolor($actual, $normal)
+
+/**
+ * Determine if a new record has been set or tied for low temperatures. Returns an empty string if no record, a star icon if tied, and a filled star icon if a new record is set. Returns an empty string if either actual or record is "M".
+ * @param mixed $actual The actual observed value.
+ * @param mixed $record The record value.
+ * @return string The HTML for the record indicator.
+ */
 function new_record2($actual, $record)
 {
     if ($actual == "M" || $record == "M") return "";
     if ($actual == $record) return '<i class="bi bi-star" aria-hidden="true"></i>';
     if ($actual < $record) return "<i class=\"bi bi-star-fill\" aria-hidden=\"true\"></i>";
```

PHPStan (2.2.7)

[error] Function new_record2() should return string but return statement is missing.


(return.missing)